### ISSUE TITLE
Multiple ports forwarded in a single expose ?

### ISSUE THREAD

**Issue by footplus** [omrikiei/ktunnel#43]
Hello,

First, thanks for writing this software, which is of great use to me.

I'm trying to use ktunnel to expose a locally running service (actually an Apache Spark driver that need to talk to Apache Spark Executors it spaws inside the cluster) that happens to listen on several ports at the same time.

What i'm basically looking to do is a multiport remote service forwarded to multiple local ports.

I initially thought this was not possible, because I had a strange error message:
```
FATA[0000] Failed to expose local machine as a service: Service "my-remote" is invalid: [spec.ports[0].name: Required value, spec.ports[1].name: Required value]
```

In reality this seems to stem from the fact that the service name seem to be mandatory when having multiple services in recent Kubernetes (v.1.18 ?)

Thanks !

**Comment by footplus** [comment]
Adding a name in the spec seems to fix the multiport expose issue for me, but I don't know if this is the correct fix regarding the range of Kubernetes that needs to be supported for ktunnel.

```
diff --git a/pkg/k8s/exposer.go b/pkg/k8s/exposer.go
index 6c637e8..abf5010 100644
--- a/pkg/k8s/exposer.go
+++ b/pkg/k8s/exposer.go
@@ -3,6 +3,7 @@ package k8s
 import (
        "context"
        "errors"
+       "fmt"
        "github.com/omrikiei/ktunnel/pkg/common"
        log "github.com/sirupsen/logrus"
        v12 "k8s.io/api/core/v1"
@@ -33,6 +34,7 @@ func ExposeAsService(namespace, name *string, tunnelPort int,
                }
                ports[i] = v12.ServicePort{
                        Protocol: protocol,
+                       Name:     fmt.Sprintf("port-%d", i),
                        Port:     parsed.Source,
                        TargetPort: intstr.IntOrString{
                                Type:   intstr.Int,
```

**Comment by omrikiei** [comment]
fix merged to v1.3.6, thank you @footplus
